.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Create Release Pull Request'
+
+on:
+  create:
+    branches:
+      - releases/*
+
+jobs:
+  build:
+    name: Create Release Pull Request
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v2
+        if: ${{ !contains(github.ref, 'test') }}
+
+      - name: Create Pull Request content
+        if: ${{ !contains(github.ref, 'test') }}
+        run: |
+          PR_TITLE=`./script/draft-release/release-pr-content.sh title ${GITHUB_REF#refs/heads/}`
+          PR_BODY=`./script/draft-release/release-pr-content.sh body ${GITHUB_REF#refs/heads/}`
+
+          echo "PR_BODY<<EOF" >> $GITHUB_ENV
+          echo "$PR_BODY" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
Since the body will have multiple lines, I found I had to do this dance to store it in an env variable 😅